package.json

```diff
@@ -2,7 +2,7 @@
   "name": "quoting-service",
```

Comment:
We are missing a couple of things here...

Npm scripts `replace`, as well as the `standard-version` config overrides, and standardization of the test npm scripts.

Reply:
added, ready for final review